content/vendor/helm/helm-processing.md

```diff
@@ -1,23 +1,110 @@
 
-Except in the case of [Helm as KOTS application airgap installations](/vendor/helm/helm-airgap-builder), the processing of the chart is managed in the end customer environment (via Kots CLI or as part of the Admin Console). 
-This means that the customer supplied values, license values and existing values can be used to create the deployable manifests. In either scenario the resulting deployment is comprised of raw Kubernetes manifests. 
-By doing this, cluster operator's are always able to view the exact diff between what is currently deployed and what the update will deploy. 
-This level of change management provides the necessary transparency to provide the level of assurance that cluster operators require.
+Except in the case of [Helm as KOTS application airgap installations](/vendor/helm/helm-airgap-builder), the processing of the chart is managed in the end customer environment (via Kots CLI or as part of the Admin Console). This means that the customer supplied values, license values and existing values can be used to create the deployable manifests. In either scenario the resulting deployment is comprised of raw Kubernetes manifests. By doing this, cluster operator's are always able to view the exact diff between what is currently deployed and what the update will deploy. This level of change management provides the necessary transparency to provide the level of assurance that cluster operators require.
 
 ## Helm V3
 
-KOTS will check the apiVersion supplied in the Chart.yaml file of the Helm Chart to check if Helm V3 is necessary. By default (if Chart.yaml is not supplied or apiVersion is not present), KOTS will use Helm V2 to process all Helm Charts to create deployable YAML.
-Optionally, in the KOTS [`HelmChart`](https://kots.io/reference/v1beta1/helmchart/) resource an API version may be specified. When the `helmVersion` property is set to `"v3"`, KOTS will use Helm V3 to process the Helm Chart.
+KOTS will check the apiVersion supplied in the Chart.yaml file of the Helm Chart to check if Helm V3 is necessary. By default (if Chart.yaml is not supplied or apiVersion is not present), KOTS will use Helm V2 to process all Helm Charts to create deployable YAML. Optionally, in the KOTS [`HelmChart`](https://kots.io/reference/v1beta1/helmchart/) resource an API version may be specified. When the `helmVersion` property is set to `"v3"`, KOTS will use Helm V3 to process the Helm Chart.
 
-Helm V3 integration is currently a beta feature. 
-In the future, KOTS may be updated to use Helm V3 by default.
+In the future KOTS may be updated to use Helm V3 by default.
```

Comment:
This was already on the page before I added my content.  I agree it's probably not worth stating here, I think we've worked past this at a technical level by automatically detecting the chart version anyway.

Since it has already been on the page for some time, are we OK to remove this, or does that become another discussion?

Reply:
Do we explicitly state that we use v2 by default? If not, we should. This alludes to that fact also it may be unexpected that this is the case since v2 is EOL (I am presuming we have not changed because it could be a breaking change?)

Reply:
technically, if left undefined and on a version of KOTS later than 1.42 or something, we will auto-detect the Helm version and use what the chart requires.  So the real `default` is "we'll figure it out and use the right one"

Reply:
Maybe this is the place to mention that, though we should be specific about detecting v2 and v3 only (IE to pre-empt next version of chart when v4 comes out).
Should we do one of those note/callouts about the version where this changed in KOTS as well and allude to old behavior too?

Reply:
I reviewed another PR recently from Todd where he was updating some Helm pages and Vera had called out that V2 was used by default and we ensured that was stated in that doc. I'd have to go search, but we did state that somewhere in the helm docs.